devtools/conda-envs/rdkit.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
   - openmm >=7.6
   - openff-forcefields >=2023.05.1
   - smirnoff99Frosst
-  - openff-units =0.2.0
+  - openff-units
```

Comment:
This is still pinned in `openeye.yaml` and other test envs. Should it be removed from all of them?

Reply:
Probably